modules/rhcos-load-firmware-blobs.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+[id="rhcos-load-firmware-blobs_{context}"]
+
+= Loading custom firmware blobs in the machine config manifest
+
+Because the default location for firmware blobs in `/usr/lib` is read-only, you can locate a custom firmware blob by updating the search path. This enables you to load local firmware blobs in the machine config manifest when the blobs are not managed by {op-system}.
+
+.Procedure
+
+. Create a Butane config file, `98-worker-firmware-blob.bu`, that updates the search path so that it is root-owned and writable to local storage. The following example places the custom blob file under `/var/lib/firmware`.
++
+[NOTE]
+====
+See "Creating machine configs with Butane" for information about Butane.
+====
+.Butane config file for custom firmware blob
++
+[source,yaml]
+----
+variant: openshift
+version: 4.9.0
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 98-worker-firmware-blob
+storage:
+  files:
+  - path: /var/lib/firmware/<package_name> <1>
+    contents:
+      local: local-<package_name> <2>
+    mode: 0644 <3>
+spec:
+  kernelArguments:
+    - 'firmware_class.path=/var/lib/firmware' <4>
+----
++
+<1> Set the local storage path for storing the firmware package in your firmware root.
+<2> Specifies a file with contents that are read from a local file directory on the system running Butane. The path of the local file is relative to a `files-dir` directory, which must be specified by using the `--files-dir` option with Butane in the following step.
```

Comment:
You can drop the `local-` prefix on the filename; it should just be the name of the file on your local host.

Reply:
Also dropped the `local-` prefix from the filename on line 49:

> $ butane 98-worker-firmware-blob.bu -o 98-worker-firmware-blob.yaml --files-dir <package_name>